specs/architecture.html

```diff
@@ -348,7 +348,7 @@
       	  <tr>
       	    <td>Payment App</td>
       	    <td>Digital Wallet, Payment Instrument </td>
-      	    <td>A piece of software to pay</td>
+      	    <td>A piece of software to manage payment methods and coordinate the issuance of payments.</td>
       	  </tr>
```

Comment:
A payment app does not manage payment methods. It supports a specific set of payment methods of which only a subset may be enabled because the user may have only configured the app with the necessary information (like payment instrument credentials) for certain methods.

I also don't like the wording "issuance of payments".

Proposed change:
```
<td>An application, registered by the payer, that supports 
processing payment requests and returning payment responses 
for one or more payment methods.</td>
```